build.sh

```diff
@@ -20,35 +54,82 @@ show_files_with_size () {
         echo "${FILE_SIZE}"
     done
 }
-#########################
-# variables 
-#########################
 
-echo $0 
+build () {
+    
+    echo -e  INFO: "${CINFO} Cleaning the dependencies ... ${CRESET}"
+    yarn clean
 
-echo "Cleaning the dependencies ..."
-yarn clean
+    echo -e  INFO: "${CINFO} Adding dependencies ... ${CRESET}"
+    yarn install
 
-echo "Adding dependencies ..."
-yarn install
+    echo -e  INFO: "${CINFO} Creating production build ... ${CRESET}"
+    yarn build
 
-echo "Creating production build ..."
-yarn build
+    echo -e INFO: "${CINFO} Build distribution contents  ... ${CRESET}"
+    show_files_with_size
 
-echo "Build distribution contents  ..."
-show_files_with_size
+    T_SIZE=$(du -hs dist)
+    echo -e INFO " ${CWARN} Size: ${T_SIZE} ${CRESET}" 
+
+    echo -e INFO: "${CINFO} Removing unwanted contents ... ${CRESET}"
+    remove_unwanted_files
+
+    T_SIZE=$(du -hs dist)
+    echo -e INFO: "${CINFO} Deployable dist size ${CWARN}  ${T_SIZE} ${CRESET}" 
+
+    RELEASBLE_BUILD="foglamp-gui-${__version__}.tar.gz"
+    echo -e  INFO: "${CINFO} Creating compressed build artifacts for release ... ${CRESET}"
+    tar -zcvf ${RELEASBLE_BUILD} dist &>/dev/null
+    echo "Created ${RELEASBLE_BUILD}"
+    echo -e INFO: "${CINFO} Done. ${CRESET}"
+
+}
+
+############################################################
+# Execute the command specified in $OPTION
+############################################################
+execute_command() {
+
+  if [ "$OPTION" == "HELP" ]
+  then
+    echo "${USAGE}"
+
+  elif [ "$OPTION" == "VERSION" ]
+  then
+    echo $__version__
+
+  elif [ "$OPTION" == "START" ]
+  then
+    build
+
```

Comment:
Remove `START`?

Reply:
no, we are setting this as default option